cmd/podman/validate/args.go

```diff
@@ -50,34 +50,30 @@ func IDOrLatestArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-// TODO: the two functions CheckAllLatestAndCIDFile and CheckAllLatestAndPodIDFile are almost identical.
-//       It may be worth looking into generalizing the two a bit more and share code but time is scarce and
-//       we only live once.
-
-// CheckAllLatestAndCIDFile checks that --all and --latest are used correctly.
-// If cidfile is set, also check for the --cidfile flag.
-func CheckAllLatestAndCIDFile(c *cobra.Command, args []string, ignoreArgLen bool, cidfile bool) error {
+// CheckAllLatestCIDAndPodIDFile checks that --all and --latest are used correctly for containers and pods
+// If idFileFlag is set is set, also checks for the --cidfile or --pod-id-file flag.
+func CheckAllLatestCIDAndPodIDFile(c *cobra.Command, args []string, ignoreArgLen bool, idFileFlag string) error {
```

Comment:
Nit: CheckAllLatestIDFile(...)
Is easier to read and type.

Reply:
I don't know if we care about API Compliance for these files, but perhaps the old function calls should remain and call into this new function?

Reply:
Fixed